environment.yml

```diff
@@ -22,6 +22,7 @@ dependencies:
 - conda-forge::gxx_linux-64
 - conda-forge::rpy2=2.9.4
 - conda-forge::libiconv=1.15
+- conda-forge::r-biocmanager
```

Comment:
I am adding this package so we don't have to worry about whether `BiocManager` is installed or not in R sessions.

Reply:
Does this mean I will not need to have these cells in my analysis notebooks:
```
#if (!requireNamespace("BiocManager", quietly = TRUE))
#    install.packages("BiocManager")
#BiocManager::install("derfinder")
#BiocManager::install("recount")
```

That would be nifty

Reply:
These two lines
```
#if (!requireNamespace("BiocManager", quietly = TRUE))
#    install.packages("BiocManager")
```
won't be needed.

For `derfinder` (line 3), we still need to install it in conda's `environment.yml` file (if that package is available in conda). I will take care of that in later PR(s). 

For `recount` (line 4), see my comments below.